Helper/Data.php

```diff
@@ -43,6 +42,10 @@ class Data extends AbstractHelper
 
     const CONFIG_CC_OPEN_ONCTRACT = 'payment/reach_cc/allow_open_contract';
 
+    CONST DHL_PRICING_STRATEGY_PATH = "payment/reach_payment/reach_dhl/pricing_strategy";
```

Comment:
just curious, is there a reason these CONST's are all caps? The other ones aren't

Reply:
Fixed 🙃